### PR TITLE
Adding TREZOR to providers

### PR DESCRIPTION
**Pull request by peca23**
TREZOR works as U2F token

### CODE REVIEW

File: _data/devices/providers.yml [hunk 1]
```diff
@@ -173,6 +173,11 @@ providers:
 #    img: transakt.png
 #    otp: Yes
 
+  - name: TREZOR
+    url: https://www.trezor.io
+    img: trezor.png
+    u2f: Yes
```
Comment (prusnak) [inline _data/devices/providers.yml:180]: `hardware: Yes` is missing

Comment (peca23) [reply _data/devices/providers.yml:180]: Thank you! Added..

**Comment by peca23** [comment]
Any news on this?

**Comment by peca23** [comment]
Thanks!

**Pull request merged**